cnxdb/contrib/testing.py

```diff
@@ -30,15 +38,21 @@ def get_settings():
 
 
 def is_venv():
-    """Returns a boolean telling whether the application is running
-    within a virtualenv (aka venv).
+    """Tells whether the application is running within a virtualenv
+    (aka venv).
+
+    :rtype: bool
```

Comment:
Oh, first step towards type hints, eh?

Reply:
Not programmatically enforced, but it's useful in the documentation